README.md

```diff
@@ -21,8 +21,7 @@ To install from source, follow the instructions in the [documentation](https://q
 
 Now that Qiskit is installed, it's time to begin working with Terra.
 
-We are ready to try out a quantum circuit example, which is simulated locally using
-the Qiskit BasicAer element. This is a simple example that makes an entangled state.
+We are ready to try out a quantum circuit example, which is simulated locally using Qiskit's `BasicAer`. This is a simple example that makes an entangled state.
```

Comment:
If we're going to mention basicaer here I would call out that it's very limited and different from qiskit-aer. If you want to do real simulation work you should use qiskit-aer as basicaer is just for testing.

Reply:
done in 7055ed04ad0746801d4236a03ac545d2b5ee3774